src/StreamingDataFrames/requirements.txt

```diff
@@ -1,2 +1,3 @@
 confluent-kafka>=2.2,<2.3
-requests>=2.28
+requests>=2.28,<2.29
```

Comment:
Hey @tim-quix , could you pls provide some details on why we need `requests` to be `2.28` here? 

I relaxed the `requests` version specifically here because it's a widely used library, and other projects might already have it in their requirements. 
Otherwise, there might be either dependency resolution conflicts or we just enforce old version of `requests` for no reason.